src/utils/looper.ts

```diff
@@ -42,18 +48,17 @@ async function doProbes(config: Config) {
         }
       })
     } catch (error) {
-      log(
-        'id:',
-        item.id,
-        '- status: ERROR for:',
-        item.request.url,
-        ':',
-        error.message
-      )
+      if (error) {
```

Comment:
How come there is no error inside this catch?

Reply:
fixed.